frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsListComponent.tsx

```diff
@@ -52,73 +53,77 @@ const FieldsListComponent: React.FC<FieldListProps> = ({ farmDetails }) => {
           <StyledCommentContainerMobile>
             <StyledListItem width="100%">
               <h2>Field Comments (optional)</h2>
-              <p key={fields.Comment}>{fields.Comment}</p>
+              <p key={field.Comment}>{field.Comment}</p>
             </StyledListItem>
           </StyledCommentContainerMobile>
-          <StyledListItemGroupContainer>
-            <StyledListItem width="300px">
-              <h2>Lab (Soil Test Methods)</h2>
-              <p key={fields.SoilTest.TestingMethod}>
-                {fields.hasSoilTest ? fields.SoilTest.TestingMethod : ComponentText.NA}
-              </p>
-            </StyledListItem>
-            <StyledListItem width="180px">
-              <h2>Sampling Month</h2>
-              <p key={fields.SoilTest.sampleDate}>
-                {fields.hasSoilTest ? fields.SoilTest.sampleDate : ComponentText.NA}
-              </p>
-            </StyledListItem>
-            <StyledListItemGroup>
-              <StyledListItem width="180px">
-                <div className="smallItems">
-                  <h2>NO3-N (ppm)</h2>
-                  <p key={fields.SoilTest.valNO3H}>
-                    {fields.hasSoilTest ? fields.SoilTest.valNO3H : ComponentText.NA}
-                  </p>
-                </div>
+
+          {field.HasSoilTest && (
```

Comment:
Remove this line here and its closing so the Test Methods will still show

Reply:
The problem is the keys accessing the values that might not exist in .nmp files, such as TestingMethod. This crashes the application because it's trying to access null values. I'll fix it, this was just a simpler fix though and the use is already warned about the default values.

Reply:
This was actually a discussion that I had with the designer, even if the soiltest or leaftest are false, it should still display with the default values mentioned.

Reply:
This was more work then I predicted, but it's done now.